src/app/views/dashboard/dashboard.html

```diff
@@ -15,12 +15,7 @@ <h3 class="text-center page-header">{{ $ctrl.currentProject.projectTitle }} Over
     <div class="row">
       <div class="col-sm-4 col-md-3 col-lg-2"><strong>Project link:</strong></div>
       <div class="col-sm-8 col-md-9 col-lg-10">
-        <a
-          href="https://geome-db.org/workbench/overview?projectId={{
-            $ctrl.currentProject.projectId
-          }}"
-          >https://geome-db.org/workbench/overview?projectId={{ $ctrl.currentProject.projectId }}</a
-        >
+        <a href="{{ $ctrl.url }}">{{ $ctrl.url }}?projectId={{ $ctrl.currentProject.projectId }}</a>
```

Comment:
I don't think this will work. Need the projectId in the href

Reply:
It technically works because the only time the user clicks on the link, they're already on the page, with the project they want to look at cached in their browser. 
I can add it to the href like i did to the visual text, but we get this weird functionality where the url keeps concatenating, ends up looking like this: http://localhost:3000/workbench/overview?projectId=%7B%7B$ctrl.currentProject.projectId%3FprojectId%3D%7B%7B$ctrl.currentProject.projectId

Reply:
the problem is, this is mostly for sharing with others. If I right click and "copy link address", the href will be copied.

That is happening b/c you use $location.absUrl. You should just get the url w/o the path & manually set the path in the html page

Reply:
Ok, got it

Reply:
lmk when you make the change and I'll merge this

Reply:
Ok i've fetched absUrl and split out the path, and manually set path in html for both text and href this should work

Reply:
👍